packages/suite/src/components/wallet/PrivacyAccount/MaxMiningFeeSetup.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { Translation } from '@suite-components';
+import { useDispatch } from '@suite-hooks';
+import { useTheme } from '@trezor/components';
+import { coinjoinAccountUpdateMaxMiningFee } from '@wallet-actions/coinjoinAccountActions';
+import { SetupSlider } from './SetupSlider';
+
+const unit = 'sat/vB';
+const labels = [1, 250, 500].map(number => ({
+    value: `${number} ${unit}`,
+}));
+
+interface MaxMiningFeeSetupProps {
+    accountKey: string;
+    maxMiningFee: number;
+}
+
+export const MaxMiningFeeSetup = ({ accountKey, maxMiningFee }: MaxMiningFeeSetupProps) => {
+    const dispatch = useDispatch();
+
+    const theme = useTheme();
+
+    const updateMaxMiningFee = (value: number) => {
+        dispatch(coinjoinAccountUpdateMaxMiningFee(accountKey, value));
+    };
+
+    const trackStyle = {
+        background: `\
+            linear-gradient(270deg,\
```

Comment:
Not necessary.. Max mining fee doesn't mean you will pay such a big amount..
Actually I'm thinking about reverting the red/green... because if you select too low `max mining fee` you will be missing some rounds..

Reply:
@hynek-jina Shouldn't it go from red to greed to red then?

Reply:
Yes.. but most of it should be green.. so the red could be only in small area (for example `high priority` should be already nicely green)

Reply:
I removed the red gradient after discussion with @hynek-jina. No need to warn about high fees, preference should be given to warning about insufficient fee limit.